index.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const path = require('path')
+
```

Comment:
['use strict'] \n Core Modules \n Lib Modules, but that's entirely personal convention.